bedita-app/views/elements/filters.tpl

```diff
@@ -12,29 +12,35 @@
 *}
 
 {*
-	- if you want to customize the label for categories pass an array('label' => 'mylabel') instead of true
-	- if you want to customize the label for statuses pass an array('on' => 'myOnLabel', 'draft' => 'myDraftLabel, 'off' => 'myOffLabel) instead of true
-	  [if you don't want to see one of the status from the filer menu don't put it in the array]
+    - if you want to customize the label for categories pass an array('label' => 'mylabel') instead of true
+    - if you want to customize the label for statuses pass an array('on' => 'myOnLabel', 'draft' => 'myDraftLabel, 'off' => 'myOffLabel) instead of true
+      [if you don't want to see one of the status from the filer menu don't put it in the array]
+
+    - 'append' is used to add custom HTML 'before' and/or 'after' core filters.
 *}
 
 {$defaultOptions = [
-		'word' => true,
-		'tree' => true,
-		'treeDescendants' => true,
-        'relations' => true,
-		'type' => false,
-		'language' => true,
-		'customProp' => ['showObjectTypes' => false],
-		'categories' => true,
-		'mediaTypes' => false,
-		'url' => $html->url("/")|cat:$view->params.controller|cat:'/'|cat:$view->params.action,
-		'tags' => true,
-		'status' => false
-	]
-}
+    'word' => true,
+    'tree' => true,
+    'treeDescendants' => true,
+    'relations' => true,
+    'type' => false,
+    'language' => true,
+    'customProp' => ['showObjectTypes' => false],
+    'categories' => true,
+    'mediaTypes' => false,
+    'url' => $html->url("/")|cat:$view->params.controller|cat:'/'|cat:$view->params.action,
```

Comment:
[very minor] Maybe `$html->url(['controller' => $view->params.controller, 'action' => $view->params.action])` would work too.

Reply:
Yes it shoud. It was an old thing... Fix is on the way